xapian-core/tests/harness/backendmanager_honey.cc

```diff
@@ -88,6 +88,28 @@ BackendManagerHoney::get_alt_writable_database(const string & name,
     return wdb;
 }
 
+Xapian::WritableDatabase
+BackendManagerHoney::get_generated_database(const string& name)
+{
+    return get_alt_writable_database(name, string());
```

Comment:
This is the only call to `get_alt_writable_database()` so we can move the code from that here and eliminate that entirely.